pkg/auth/tokenparser_test.go

```diff
@@ -8,67 +8,77 @@ import (
 	"github.com/codeready-toolchain/registration-service/pkg/auth"
 	"github.com/codeready-toolchain/registration-service/pkg/configuration"
 	"github.com/codeready-toolchain/registration-service/test"
+	authsupport "github.com/codeready-toolchain/toolchain-common/pkg/test/auth"
 
 	"github.com/dgrijalva/jwt-go"
 	uuid "github.com/satori/go.uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 )
 
-func TestTokenParser(t *testing.T) {
+type TestTokenParserSuite struct {
```

Comment:
just curious, why do you want to use a `UnitTestSuite` here, instead of the standalone tests?

Reply:
I was assigned a story awhile ago to change all the tests to use the suite instead. I guess this test slipped in afterwards. When i was fixing the tests, i thought to make it follow the convention we agreed on. 

The PR for test suite is here: https://github.com/codeready-toolchain/registration-service/commit/7bce17cec472137076dcdb2d09b53ca8683f8ac4

Reply:
ok, I see. To be honest, I'm not sure there's much value in using a `UnitTestSuite` here, but it's ok to have it, and it's probably more consistent with the other tests.